frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -249,6 +250,42 @@ export const createRoute = (
   return k8sCreate(RouteModel, route, dryRun ? dryRunOpt : {});
 };
 
+export const createResourceLimit = (
+  formData: GitImportFormData,
+  imageStream: K8sResourceKind,
+  dryRun: boolean,
+): Promise<K8sResourceKind> => {
+  const {
+    name,
+    project: { name: namespace },
+    application: { name: application },
+    labels: userLabels,
+    limits: resource,
+  } = formData;
+
+  const defaultLabels = getAppLabels(name, application, imageStream.metadata.name);
+
+  const resourceLimit = {
+    kind: 'ResourceQuota',
+    apiVersion: 'v1',
+    metadata: {
+      name,
+      namespace,
+      labels: { ...defaultLabels, ...userLabels },
+    },
+    spec: {
+      hard: {
+        'limits.cpu': `${resource.cpu.limit + resource.cpu.limitUnit}`,
+        'limits.memory': `${resource.memory.limit + resource.memory.limitUnit}`,
+        'requests.cpu': `${resource.cpu.request + resource.cpu.requestUnit}`,
+        'requests.memory': `${resource.memory.request + resource.memory.requestUnit}`,
```

Comment:
Several instances where you could concatenate using the following method instead:
```suggestion
        'requests.memory': `${resource.memory.request}${resource.memory.requestUnit}`,
```

Reply:
Done.